packages/big-design/src/components/Table/Actions/Actions.tsx

```diff
@@ -70,7 +111,8 @@ export const Actions = memo(
         padding="small"
```

Comment:
🍹Could you try and remove the `style={{ marginLeft: 'auto' }}` on the pagination, and add `justifyContent="space-between"` to `StyledActions`? We should try and clean up those `style` attributes.

Reply:
In this case, we have multiple actions, one of them being the `itemName` so if we do `space-between` we won't be getting what we want :(

However, I can move this to a custom-styled Flex.

We should add `auto` to `MarginProps`

Reply:
Ahh yeah, well in that case maybe you could change the `Flex.Item` for both SelectAll and ItemName to have `alignSelf="flex-start"` and Pagination `alignSelf="flex-end"`? 🤷‍♀️